english/triathlon/triathlon.md

```diff
@@ -0,0 +1,8 @@
+You were always intrigued my endurance sports. 
```

Comment:
@alexpaul, the point is that if a reader starts in `language.md` and keeps clicking links, they will never reach this change. You should fix this by finding a place in the existing story and adding an option that leads to your page.